publishable/config/voyager.php

```diff
@@ -94,21 +95,19 @@
     */
 
     'widgets' => [
-        [
-            'name'  => 'User',
+        'user' => [
             'icon'  => 'voyager-group',
             'model' => TCG\Voyager\Models\User::class,
             'url'   => 'admin/users',
             'image' => '/images/widget-backgrounds/02.png',
         ],
```

Comment:
Problem with that is that the view for it can't be customized for each widget. Which it can now using the class handlers.